Makefile

```diff
@@ -9,7 +9,7 @@ AP ?= ansible-playbook -vv -c local -i localhost, -e ansible_python_interpreter=
 PATH_TO_SECRETS ?= $(CURDIR)/secrets/
 COV_REPORT ?= term-missing
 COLOR ?= yes
-SOURCE_BRANCH ?=
+SOURCE_BRANCH ?= $(shell git branch --show-current)
```

Comment:
actually, now that I think about this, I second Hunor, shouldn't we hardcode "master" here?

will the build even work if it's something which is not stable nor master?

Reply:
https://github.com/packit-service/packit-service/blob/master/files/tasks/process-source-branch.yaml#L9

Oh come on guys, this is a hotfix PR, not a new feature PR :-)
Let me merge this and leave this to @sakalosj :-)